### DISCUSSION TITLE
What to do when I have multiple occupancies?

### DISCUSSION THREAD

**Discussion by mysecondnameisdanger** [Electrostatics/pdb2pqr#390] [Q&A]
It is frequent to have multiple rotamers/conformations/alternative locations for some residues in a pdb file.
If I upload my pdb with both conformers to the pdb2pqr/apbs web server, one gets removed (I guess you only keep the first conformer and delete all others?)...
`WARNING:main.py:424:setup_molecule:Multiple occupancies found in ASP A 18. At least one of the instances is being ignored.`

What if the occupancies are 50% for conformer A and 50% for conformer B? should not pdb2pqr take into account that both scenarios can occur? Where I can find documentation on this?

Thanks

**Comment by sobolevnrm** [comment]
Hello.  I believe PDB2PQR just takes the first conformer right now -- it does not accommodate both conformations.
